pkg/virt-launcher/virtwrap/manager_test.go

```diff
@@ -1622,6 +1623,17 @@ var _ = Describe("Manager", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(domSpec).ToNot(BeNil())
 		})
+
+		Context("on call to InterfacesStatus", func() {
+			It("should return nil when no interfaces exists", func() {
+				agentStore := agentpoller.NewAsyncAgentStore()
+				manager, _ := NewLibvirtDomainManager(mockConn, testVirtShareDir, nil, 0, &agentStore, "/usr/share/OVMF")
+				libvirtmanager := manager.(*LibvirtDomainManager)
+
+				interfaces := libvirtmanager.InterfacesStatus(nil)
+				Expect(interfaces).To(BeNil())
+			})
+		})
```

Comment:
I'm missing here coverage for `GetGuestOSInfo` and `InterfacesStatus` with a non empty domain interface list as an argument.

Reply:
GetGuestOSInfo with info is covered in the resync test, unless it needs a test here as well 
see please comment
https://github.com/kubevirt/kubevirt/pull/5560#discussion_r626284924